WalletWasabi.Gui/Tabs/SettingsView.xaml

```diff
@@ -65,30 +65,43 @@
                 <ToggleButton IsChecked="{Binding LurkingWifeMode}" Content="{Binding LurkingWifeMode, Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" Command="{Binding LurkingWifeModeCommand}" />
                 <TextBlock VerticalAlignment="Center">Lurking Wife Mode hides sensitive content.</TextBlock>
               </StackPanel>
+              <controls:GroupBox Title="Screen Lock" TextBlock.FontSize="16" Padding="10" Margin="0 5 10 5">
+                <StackPanel Orientation="Vertical" Spacing="5">
+                  <StackPanel Margin="0 10" Orientation="Horizontal" Spacing="5">
+                    <ToggleButton IsChecked="{Binding EnablePINEntry}" Content="{Binding EnablePINEntry,  Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" Command="{Binding EnablePINLockCommand}" />
+                    <TextBlock VerticalAlignment="Center">Use PIN to unlock the screen.</TextBlock>
+                  </StackPanel>
+                  <StackPanel Orientation="Vertical" Spacing="5">
+                    <controls:NoparaPasswordBox Watermark="PIN" Password="{Binding OldPINPwdBox}" UseFloatingWatermark="True" />
```

Comment:
Use `FixedPasswordText` to not confuse the user that the text is different.

Reply:
I replaced `Watermark` with that and it doesnt seem to be working?

Reply:
Besides, I haven't seen it being used elsewhere in the app so it might cause some UX inconsistency too.